tests/test_operations.py

```diff
@@ -36,6 +36,9 @@ def mocked_env(mocker):
     mock_env.blueprint_dir = BLUEPRINT_DIR
     mock_env.plugin_manager = PLUGIN_MANAGER
     mock_env.service_template_name = SERVICE_TEMPLATE_NAME
+    mock_env.model_storage.service_template.list = mocker.MagicMock(
```

Comment:
explain why this line is needed

Reply:
no need for creating another mock. Just change the return value attribute of list